resources/js/main.js

```diff
@@ -0,0 +1,287 @@
+import ithenticateSimilarityScoreCell from "./Components/ithenticateSimilarityScoreCell.vue";
+
+pkp.registry.registerComponent("ithenticateSimilarityScoreCell", ithenticateSimilarityScoreCell);
+
+const {useLocalize } = pkp.modules.useLocalize;
```

Comment:
I see that the formatting is bit all over the place. I have default setting in VS code to use prettier on save when is available. And you have that in the dependencies there.

But maybe I should add formatting command to the backendUiExample? Opened to ideas how to make it more intuitive for developers who don't have these things setup by default..